docs/partials/_quickstart-prereqs.md

```diff
@@ -0,0 +1,10 @@
+## Prerequisites
+
+* A Rancher server (v2.7.0) configured (server-url set)
```

Comment:
nit: the latest UI with the latest ISO build feature requires Rancher 2.7.2 (which supports the latest version of the rancher ui extension operator, needed by the elemental UI extension 1.2.0).

Reply:
Will be covered in a follow-up PR